.github/workflows/ci.yml

```diff
@@ -132,9 +132,12 @@ jobs:
             matrix=$(
               yq --output-format json <<< "${matrix}" | jq '[.[] | select('"${query}"')]'
             )
-          else
+          elif ${{ !contains(github.event.pull_request.labels.*.name, 'no-ci-targets' ) }}; then
             echo 'Running all targets.'
             matrix="$(yq --output-format json <<< "${matrix}")"
+          else
+            echo 'No CI targets will be used.'
+            matrix='[]'
```

Comment:
I think it's better to have this as a `if:` on the `build` matrix.

Reply:
That sounds better: we don't need `try --target` and the `no-ci-targets` to ever coexist.